recipes/python-suitesparse-graphblas/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "python-suitesparse-graphblas" %}
```

Comment:
I find it annoying when Python packages are named `python-*`.  However, in this case, I expect this package to always be used as a dependency of another package, so users won't generally need to do e.g. `conda install python-suitesparse-graphblas`.  This name in fact makes it clear that these are Python bindings.

Reply:
What name will this package be on PyPI?  `suitesparse-graphblas` (which is the name of the package) or `python-suitesparse-graphblas`?

Reply:
I would be okay with suitesparse-graphblas because PyPI is *only* python packages, so there is no potential confusion.

Reply:
`python-suitesparse-graphblas` works for me.